SSDynamicText/UIView+SSTextSize.h

```diff
@@ -27,28 +27,35 @@ typedef void (^SSTextSizeChangedBlock) (NSInteger);
  * When the user changes her preferred text size, the callback block is called with the
  * new text size delta from the default.
  */
-- (void) ss_startObservingTextSizeChangesWithBlock:(SSTextSizeChangedBlock)block;
+- (void)ss_startObservingTextSizeChangesWithBlock:(SSTextSizeChangedBlock)block;
 
 /**
  * Stop observing changes to text size.
  */
-- (void) ss_stopObservingTextSizeChanges;
+- (void)ss_stopObservingTextSizeChanges;
 
 /**
  * Force a recalculation of our preferred text size.
  */
-- (void) preferredContentSizeDidChange;
+- (void)preferredContentSizeDidChange;
 
 /**
  * Default FontName if set in Info.plist or systemFontName if not set
  * Key: kSSDynamicDefaultFontName
  */
-- (NSString *) ss_defaultFontName;
+- (NSString *)ss_defaultFontName;
```

Comment:
I merge this PR, and I'll create new PR over a days with generics, nullability, this property and some minors improvements ;)